polyfill/promise.ts

```diff
@@ -1,5 +1,5 @@
 //@ts-ignore
-import Promise from "promiscuous";
+import Promise from "promise-polyfill";
```

Comment:
This seems to be the wrong file to import

Remove all the code below and just do that:

```suggestion
import "promise-polyfill/src/polyfill";
```